superbol/blackbody.py

```diff
@@ -43,7 +41,6 @@ def bb_total_flux(self):
     Returns:
         float: The value of :math:`\\sigma \\frac{R^2}{D^2} T^4`
     """
-    temperature = u.Quantity(self.temperature, u.K)
```

Comment:
I've been investigating this module, and I think there's probably a lot of code that can get pulled out. I'm not sure this bb_total_flux function ever actually gets called.